Hindawi Publishers.js

```diff
@@ -71,6 +76,10 @@ function doWeb(doc,url)
 			ZU.processDocuments(urls, doWeb);
 		});
 	} else {
+		var namespace = {"x"  : "http://www.w3.org/1999/xhtml"}
```

Comment:
move `namespace` out of the `if` block above and reuse it here instead of re-declaring.